### PR TITLE
Credential status id optional

### PR DESCRIPTION
**Pull request by aljones15**
Updates the credentialStatus checks for the latest in the spec such as making credentialStatus id optional, throwing if more than id is provided, and expanding credentialStatus type checks.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
 - Add a new test to ensure type URLs are treated as an unordered set.
 - Add new tests for `issuer.description` including Internationalization settings.
 - Add new tests for `issuer.id` as a url and not a url.
 - Add tests for interactions between `validFrom ` and `validUntil`.
+- Expand tests for `credentialStatus.{id, type}`.
 
 ### Fixed
 - Update statement for credentialStatus id & type test.
 - Update statement for credentialSchemas test.
+- Credential status id is now optional.
```
Comment (BigBlueHat) [inline CHANGELOG.md:16]:
```suggestion
- Update statement for `credentialStatus.id` and `type` test.
- Update statement for `credentialSchemas` test.
- `credentialStatus.id` is now optional.
```

File: tests/10-vcdm2.js [hunk 2]
```diff
@@ -432,16 +432,27 @@ describe('Verifiable Credentials Data Model v2.0', function() {
...
-          './input/credential-status-missing-id-fail.json')));
+          './input/credential-status-missing-type-fail.json')));
+        await assert.rejects(endpoints.issue(require(
+          './input/credential-status-type-nonurl-fail.json')));
+        await endpoints.issue(require(
+          './input/credential-status-ok.json'));
       });
-      reportRow('credentialStatus id property MUST be a URL which MAY be ' +
-        'dereferenced.', async function() {
+      reportRow('If present (credentialStatus id), the normative guidance ' +
```
Comment (BigBlueHat) [inline tests/10-vcdm2.js:446]:
```suggestion
      reportRow('If present (credentialStatus.id), the normative guidance ' +
```

**Review by BigBlueHat — CHANGES_REQUESTED** [review]
A few tweaks. Also, the order of tests seems fairly inconsistent relative to the order these statements appear in the spec text. Could they be placed relative to the document order? Thanks!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,17 +1,19 @@
 # w3c/vc-data-model-2.0-test-suite ChangeLog
 
-## 1.1.0 -
+## 1.1.0 - 01/11/2024
```
Comment (davidlehn) [inline CHANGELOG.md:3]:
```suggestion
## 1.1.0 - 2024-01-xx
```

Comment (BigBlueHat) [reply CHANGELOG.md:3]: There is only ISO 8601. 😉 https://xkcd.com/1179/

**Pull request merged**